schwifty/bank_registry/manual_it.json

```diff
@@ -1,666 +1,1122 @@
+		"short_name": "BANCA DI CREDITO COOPERATIVO ABRUZZESE"
+	},
+	{
+		"bank_code": "08439",
+		"bic": "CCRTIT2TCAR",
+		"country_code": "IT",
+		"name": "BANCA DI CARAGLIO, DEL CUNEESE E DELLA RIVIERA DEI FIORI - CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI CARAGLIO"
+	},
+	{
+		"bank_code": "08440",
+		"bic": "ICRAITRRCAR",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI CARATE BRIANZA SCRI",
+		"primary": true,
+		"short_name": "BCC DI CARATE BRIANZA"
+	},
+	{
+		"bank_code": "08452",
+		"bic": "ICRAITRRCK0",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI VENEZIA PADOVA E ROVIGO - BANCA ANNIA S.C.",
+		"primary": true,
+		"short_name": "BCC DI VENEZIA PADOVA E ROVIGO - BANCA ANNIA"
+	},
+	{
+		"bank_code": "08453",
+		"bic": "ICRAITRRCL0",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI MILANO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO COOPERATIVO DI MILANO"
+	},
+	{
+		"bank_code": "08454",
+		"bic": "ICRAITRRCM0",
+		"country_code": "IT",
+		"name": "CREDITO PADANO BANCA DI CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "CREDITO PADANO BANCA"
+	},
+	{
+		"bank_code": "08462",
+		"bic": "CCRTIT2TROC",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DELLA ROMAGNA OCCIDENTALE - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO COOPERATIVO DELLA ROMAGNA OCCIDENTALE"
+	},
+	{
+		"bank_code": "08473",
+		"bic": "ICRAITRRD50",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOP. DI CASTIGLIONE MESSER RAIMONDO E PIANELLA",
+		"primary": true,
+		"short_name": "BCC DI CASTIGLIONE MESSER RAIMONDO E PIANELLA"
+	},
+	{
+		"bank_code": "08487",
+		"bic": "CCRTIT2TCHE",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI CHERASCO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": ""
+	},
+	{
+		"bank_code": "08491",
+		"bic": "CCRTIT2TCIV",
+		"country_code": "IT",
+		"name": "BANCO MARCHIGIANO CREDITO COOPERATIVO",
+		"primary": true,
+		"short_name": "BANCO MARCHIGIANO"
+	},
+	{
+		"bank_code": "08514",
+		"bic": "ICRAITRREA0",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DELL'OGLIO E DEL SERIO S.C.",
+		"primary": true,
+		"short_name": "BCC DELL'OGLIO E DEL SERIO"
+	},
+	{
+		"bank_code": "08519",
+		"bic": "ICRAITRREF0",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI FANO SCRL",
+		"primary": true,
+		"short_name": "BCC DI FANO"
+	},
+	{
+		"bank_code": "08530",
+		"bic": "ICRAITRREQ0",
+		"country_code": "IT",
+		"name": "BANCA D'ALBA CREDITO COOPERATIVO SC",
+		"primary": true,
+		"short_name": "BCC D'ALBA"
+	},
+	{
+		"bank_code": "08537",
+		"bic": "ICRAITRREX0",
+		"country_code": "IT",
+		"name": "CREDITO COOPERATIVO CASSA RURALE ED ARTIGIANA DI ERCHIE - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BCC ERCHIE"
```

Comment:
should we also keep `CASSA RURALE ED ARTIGIANA DI ERCHIE`?

Reply:
same as the BCC AQUARA, it was displayed as BCC ERCHIE on their website